scripts/generate-install-yaml.sh

```diff
@@ -27,3 +27,31 @@ kubebuilder create config --controller-image "${IMAGE_NAME}" \
 # Increase memory request and limit to avoid OOM issues.
 sed -i 's/memory: 20Mi/memory: 64Mi/' "${INSTALL_YAML}"
 sed -i 's/memory: 30Mi/memory: 128Mi/' "${INSTALL_YAML}"
+
+# Remove namespace resource from the top of the file
+sed -i -e '/---/,$!d' "${INSTALL_YAML}"
+
+# Remove namespace fields
+sed -i -e '/^  namespace: federation-system$/d' "${INSTALL_YAML}"
+
+if [ "${NS}" ]; then
```

Comment:
Please define this at the top of the script and set a default, as per the example of `INSTALL_YAML` and `IMAGE_NAME`, to ensure that this script outputs a reasonable default when called directly.  Also, avoid an abbreviation like `NS` for a variable that is passed in from the environment rather than used internally (i.e. use `FEDERATION_NAMESPACE` as the name of the environment variable as per the deployment script).

Reply:
Will fix in update.

Reply:
@marun I think https://github.com/kubernetes-sigs/federation-v2/blob/master/scripts/deploy-federation.sh#L50 also need update?

```
NS="${FEDERATION_NAMESPACE:-federation-system}"
```

Do you think we also need to update above as following? If so, I intend to take the re-name in another PR.

```
FEDERATION_NAMESPACE="${FEDERATION_NAMESPACE:-federation-system}"
```

Reply:
It's fine to use the `NS` name internally so long as the external name (`FEDERATION_NAMESPACE`) is verbose.